diff --git a/chrome/browser/ui/webui/about_ui.cc b/chrome/browser/ui/webui/about_ui.cc
index ee7fb0d034370939ee0c2be12a0f412c41e5ad9e..03776238fa3d6f1740a2355a1a08dda5f1fb9cf7 100644
--- a/chrome/browser/ui/webui/about_ui.cc
+++ b/chrome/browser/ui/webui/about_ui.cc
@@ -571,6 +571,7 @@ std::string ChromeURLs() {
   html += "</ul>\n";
 
   AppendFooter(&html);
+  CHROME_INTERNAL_URLS_TO_BRAVE
   return html;
 }
 
